kolibri/utils/conf.py

```diff
@@ -77,6 +77,7 @@
         "kolibri.plugins.style_guide",
         "kolibri.plugins.document_epub_render",
         "kolibri.plugins.default_theme",
+        "kolibri.plugins.oidc_provider_plugin",
```

Comment:
Do we want this to be the default during dev?

Reply:
this was supposed to be no, this was supposed to be removed